frontend/pulse3d/src/pages/uploads.js

```diff
@@ -273,36 +278,31 @@ export default function Uploads() {
   useEffect(() => {
     getAllJobs();
     // start 10 second interval
-    const uploadsInterval = setInterval(() => getAllJobs(), [1e4]);
```

Comment:
If this is getting removed in favor of another method of polling the job status, then I don't think it is working currently. If there is new code to do this, it needs to be fixed, or this should just be added back